lib/ProductOpener/KnowledgePanels.pm

```diff
@@ -1334,7 +1339,11 @@ sub create_ingredients_panel ($product_ref, $target_lc, $target_cc, $options_ref
 		title => $title,
 		ingredients_text => $ingredients_text,
 		ingredients_text_with_allergens => $ingredients_text_with_allergens,
-		ingredients_text_lc => $ingredients_text_lc,
+		ingredients_text_lc => (
```

Comment:
I'm not sure why you made that change, but it's breaking the actions in knowledge panels:

-                     "edit_field_id" : "ingredients_text_en",
+                     "edit_field_id" : "ingredients_text_English",

I will change it back.